src/SimpleTwoActionsDialog/SimpleTwoActionsDialog.js

```diff
@@ -13,63 +12,109 @@ import {
   DialogActions
 } from '@material-ui/core';
 
-const SimpleTwoActionsDialog = ({
+export const SimpleTwoActionsDialog = ({
+  id,
   open,
-  dialogTitleKey,
-  dialogBodyKey,
-  cancelLabelKey,
-  handleClickOnCancel,
-  validateLabelKey,
-  handleClickOnValidate
+  labels,
+  handleClickOnButton1,
+  handleClickOnButton2
 }) => {
-  const { t } = useTranslation();
-
   const onClose = (event, reason) => {
     if (reason !== 'backdropClick') {
-      handleClickOnCancel();
+      handleClickOnButton1();
     }
   };
 
   return (
-    <Dialog open={open} aria-labelledby="discard-changes-dialog"
+    <Dialog open={open} aria-labelledby={labels.ariaLabelledby}
       maxWidth={'xs'}
       fullWidth={true}
       onClose={onClose}
     >
-      <DialogTitle id="discard-changes-dialog-title">
-        {t(dialogTitleKey, 'Dialog title')}
+      <DialogTitle id={id + '-dialog-title'}>
+        {labels.title}
       </DialogTitle>
       <DialogContent>
-        <Typography variant='body1'>{t(dialogBodyKey, 'Dialog Body')}</Typography>
+        <Typography variant='body1'>{labels.body}</Typography>
       </DialogContent>
       <DialogActions >
         <Button
-          id="ButtonCancel"
-          onClick={handleClickOnCancel}
+          id={id + '-button1'}
+          onClick={handleClickOnButton1}
           color="primary">
-          {t(cancelLabelKey, 'Cancel label')}
+          {labels.button1}
         </Button>
         <Button
-          data-cy='dialog-discard-button'
-          id="ButtonDiscard"
-          onClick={handleClickOnValidate}
+          data-cy={id + '-button2'}
+          id={id + 'id-button2'}
```

Comment:
The id of _button2_ is not consistent with `id={id + '-button1'}` for _button1_.
Also _button2_ has a data-cy prop but _button1_ does not.